lib_mic_array/lib_build_info.cmake

```diff
@@ -0,0 +1,15 @@
+set(LIB_NAME lib_mic_array)
+set(LIB_VERSION 5.2.0)
+set(LIB_DEPENDENT_MODULES "lib_xcore_math")
```

Comment:
lib_xcore_math has been released with XCommon CMake support, so this could now be `"lib_xcore_math(2.2.0)"`. Also you could consider updating the lib_xcore_math tag in the CMakeLists.txt in the root directory of the repo.